pkg/proxy/iptables/proxier.go

```diff
@@ -1555,6 +1604,32 @@ func (proxier *Proxier) syncProxyRules() {
 		"-m", "addrtype", "--dst-type", "LOCAL",
 		"-j", string(kubeNodePortsChain))
 
+	if len(proxier.clusterCIDR) != 0 {
+		glog.Error("Should be adding the rules now")
+		writeLine(proxier.filterRules,
```

Comment:
Can you add comments to each block explaining the rationale, in as plain terms as possible.  These rules are always hard to decipher a year later.

Reply:
This one doesn't seem to need clusterCIDR to be set?